ui/src/containers/App.js

```diff
@@ -37,17 +37,22 @@ const App = () => {
     document.title = messages[language].product_name;
     if (status === 'idle') {
       dispatch(fetchConfigAction());
-      dispatch(setInitialLanguageAction());// todo removes this once the navbar provides it 
+      dispatch(setInitialLanguageAction()); // todo removes this once the navbar provides it
```

Comment:
So, should be done now? The navbar already takes care of initializing from localStorage, from what I could see in 1a1d854